src/OrbitDB.js

```diff
@@ -319,6 +336,7 @@ class OrbitDB {
     const dir = db && db.options.directory ? db.options.directory : this.directory
     await this._requestCache(address, dir, db._cache)
     this.stores[address] = db
+    this.events.emit('load', address)
```

Comment:
i believe if a relayEvent of 'load' existed then two load events would fire from this.events for the same store.

Reply:
If I remove this line and update the test like so: 
```
it('emits load event when opening database', async () => {
          let dbAddres
          const handleEvent = (addr => dbAddres = addr)
          orbitdb.events.once('load',  handleEvent)
          db = await orbitdb.open('load_event_test', { create: true, type: 'feed', relayEvents:['load'] })
          await db.load()
          assert.equal(dbAddres, db.address.toString())
      })
```

it results in a test failure:
```
1) orbit-db - Create & Open (js-ipfs)
       Open
         emits load event when opening database:
     AssertionError [ERR_ASSERTION]: undefined == '/orbitdb/zdpuAvZRphzghitg8ZbJY3hx5UeLEKPWNLam3koaMkKhKMZPZ/load_event_test'
      at Context.<anonymous> (test/create-open.test.js:397:18)
```
What you said should be true, yet it isn't. there's a flaw somewhere, but I'm not able to see if :C

Reply:
```
 if (heads.length > 0) {
      this.events.emit('load', this.address.toString(), heads)
    }
```
So load event only fires if heads is greater then zero..... idk what to do here

Reply:
Removed it for now, and fixed the tests, this may need to be revisited in the future.